server/chat-commands/info.ts

```diff
@@ -248,11 +250,11 @@ export const commands: ChatCommands = {
 		if (!targetUser || !targetUser.connected) buf += ` <em style="color:gray">(offline)</em>`;
 
 		let roomauth = '';
-		if (room?.auth && userid in room.auth) roomauth = room.auth[userid];
+		if (room?.auth.has(userid)) roomauth = room.auth.get(userid) as string;
 		if (Config.groups[roomauth] && Config.groups[roomauth].name) {
 			buf += `<br />${Config.groups[roomauth].name} (${roomauth})`;
 		}
-		const group = (Users.usergroups[userid] || '').charAt(0);
+		const group = (Users.groups.get(userid) || '').charAt(0);
```

Comment:
`Users.groups.get` should also default to `''`, which would simplify this part, too.